### PR TITLE
Fix https://github.com/istio/istio/issues/13910

### PR DESCRIPTION
**Pull request by ozevren**
This PR adds InitialWindowSize & InitialConnWindowSize parameters to both Pilot and Galley, as well as adding KeepAlive options to Galley. Combined, they both seem to highly alleviate the too many pings problem in https://github.com/istio/istio/issues/13910.

Some context here: https://github.com/grpc/grpc-go/issues/1882

### CODE REVIEW

**Comment by ozevren** [comment]
ping.

**Comment by hzxuzhonghu** [comment]
@ozevren Has this been verified?

**Comment by ozevren** [comment]
I tried in my local cluster. I appreciate independent verification.

File: galley/pkg/server/args.go [hunk 1]
```diff
@@ -49,6 +50,12 @@ type Args struct {
 	// Maximum number of outstanding RPCs per connection
 	MaxConcurrentStreams uint
 
+	// Initial Window Size for gRPC connections
+	InitialWindowSize uint
```
Comment (hzxuzhonghu) [inline galley/pkg/server/args.go]: This is for grpc stream, and `InitialConnectionWindowSize` is for a connection, should set `InitialConnectionWindowSize` bigger than `InitialWindowSize ` by default, like InitialConnectionWindowSize = 16 * InitialWindowSize ?

Comment (ozevren) [reply galley/pkg/server/args.go]: Updated.

**Comment by hzxuzhonghu** [comment]
/lgtm

Thanks  @ozevren

**Pull request merged**